docs/Brew-Test-Bot-For-Core-Contributors.md

```diff
@@ -2,25 +2,16 @@
 
 If a build has run and passed on `brew test-bot` then it can be used to quickly bottle formulae.
 
-There are two types of Jenkins jobs you will interact with:
-
-## [Homebrew Core Pull Requests](https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/)
-This job automatically builds any pull requests submitted to Homebrew/homebrew-core. On success or failure it updates the pull request status (see more details on the [main Brew Test Bot documentation page](Brew-Test-Bot.md)). On a successful build it automatically uploads bottles.
-
-## [Homebrew Testing](https://jenkins.brew.sh/job/Homebrew%20Testing/)
-This job is manually triggered to run [`brew test-bot`](https://github.com/Homebrew/homebrew-test-bot/blob/master/cmd/brew-test-bot.rb) with user-specified parameters. On a successful build it automatically uploads bottles.
```

Comment:
Do we have a replacement for this?

Reply:
Not yet!

Reply:
GitHub recently fixed the ability to use payloads in `matrix` which is good - I've not had a chance to test it out yet.

Reply:
Worst case scenario, we just copy over the `brew request-bottle` workflow from Homebrew/linuxbrew-core and target it for Snow Catalina.